### DISCUSSION TITLE
Γνωριμία

### DISCUSSION THREAD

**Discussion by epidrome** [sdy60/syllabus#2] [Show and tell]
Θα ήθελα δύο λόγια (ή ακόμη καλύτερα ένα λινκ) σχετικά με εσάς (π.χ., προσωπική σελίδα, βιογραφικό, linkedin) και αναφορικά με προηγούμενα πτυχία-εργασία-ενδιαφέροντα, καθώς και τις προσδοκίες σας από την ΣΔΥ60

Στην προσωπική σελίδα μου θα βρείτε πληροφορίες για το [βιογραφικό και τα ερευνητικά ενδιαφέροντα μου](https://vita.epidro.me/).

**Comment by dimidimots** [comment]
Καλησπέρα και από εδώ.

Η σελίδα μου στο linkedin https://www.linkedin.com/in/dimitris-dimotsis-258896222/
update 6/10 :
Προηγούμενα πτυχία πτυχίο Τει Πληροφορικής Θεσ/κης & μεταπτυχιακό Διασφάλιση Ποιότητας ΕΑΠ.
Δουλεύω ως support manager και devops στην Nomitech,

ως προς το μάθημα λόγω ότι δουλεύω σε software vendor προσδοκώ μια σχετική εξοικίωση με προγραμματισμό καθώς δεν έχω αντίστοιχες γνώσεις, καθώς και εφαρμογή διάχυτου υπολογισμού όσο το δυνατόν στη πράξη.

**Comment by afoscolou** [comment]
Θα σας στείλω στο προσωπικό σας email του ΕΑΠ ένα μίνι βιογραφικό.

**Reply by epidrome** [reply]
πως θα γίνει η γνωριμία με τους συμφοιτητές;

καταλαβαίνω ότι κάποια στοιχεία είναι ιδιωτικά, 

για αυτό η ανακοίνωση δίνει την δυνατότητα να αντιγράψετε εδώ ότι σας βολεύει

**Reply by afoscolou** [reply]
Τελείωσα το **τμήμα Φυσικής του Πανεπιστημίου Αθηνών** και συνέχισα με **μεταπτυχιακές σπουδές (MSc) στη Φυσική Υψηλών Ενεργειών στις ΗΠΑ (Michigan)**, ασχολήθηκα αρκετό διάστημα με τη διδασκαλία και γενικότερα την επιμόρφωση, κυρίως ενηλίκων, φοιτητών (British Council Athens), με την οργάνωση της διοικητικής δομής νεοφυών επιχειρήσεων με διάφορα αντικείμενα, με την εκμάθηση ξένων γλωσσών και τα ταξίδια ανά τον κόσμο. Τα τελευταία χρόνια, διαπιστώνοντας ότι οι καλπάζουσες εξελίξεις στην τεχνολογία παρουσιάζουν ιδιαίτερο ενδιαφέρον και θέτουν προκλήσεις για απόκτηση νέων γνώσεων και δεξιοτήτων, αποφάσισα να ασχοληθώ με την πληροφορική, έναν ανεκπλήρωτο στόχο από τα φοιτητικά μου χρόνια στην Αμερική. Τελειώνοντας το **μεταπτυχιακό μου στα Πληροφοριακά Συστήματα στο ΕΑΠ**, ένοιωσα περισσότερο την ανάγκη να εμβαθύνω στον τομέα των Ασύρματων Δικτύων και του ΙοΤ.  Η επιθυμία μου να παρακολουθήσω το συγκεκριμένο πρόγραμμα δεν υπαγορεύεται από βιοποριστικά κίνητρα ή επείγουσα επαγγελματική ανάγκη αλλά από την συνεχή επιδίωξή μου για προσωπική ανάπτυξη και αυτοπραγμάτωση.

**Comment by ghost** [comment]
https://europa.eu/europass/eportfolio/api/eprofile/shared-profile/addc7070-29b5-4171-adb2-6b8fcd677896?view=html

**Comment by vkaragia** [comment]
Τελείωσα την Σχολή Ναυτικών Δοκίμων/Τμήμα Μάχιμων το 2001 και έχω λάβει την εξιδείκευση Οπλισμού. Έχω υπηρετήσει για 15 έτη σε μονάδες του Αρχηγείου Στόλου (Φρεγάτες και Κανονιοφόρους) όπου εκτέλεσα τα καθήκοντα Αξιωματικού Οπλισμού, Αξιωματικού Ηλεκτρονικών Συστημάτων και Ασφάλειας Δικτύων καθώς και ως Ύπαρχος στη Φρεγάτα ΨΑΡΑ(2018-2020) και ως Κυβερνήτης στην Κανονιοφόρο ΜΑΧΗΤΗΣ(2021-2022). Έχω λάβει μεταπτυχιακό (MSc) στη Επιστήμη Υπολογιστών στις ΗΠΑ (Naval Postgraduate School, Monterey), την περίοδο 2011-2013 με υποτροφία από το Πολεμικό Ναυτικό. Έχω υπηρετήσει στη Σχολή Ναυτικών Δοκίμων ως επιτηρητής Αξιωματικός Δοκίμων και έχω διδαξει μαθήματα σχετικά με τα Οπλικά Συστήματα και τον Ηλεκτρονικό Πόλεμο. Κατά την παρούσα περίοδο υπηρετώ στο Κέντρο Επιχειρήσεων του Αρχηγείου Στόλου ως Αξιωματικός Φυλακής. Εγγράφηκα στο παρόν Μεταπτυχιακό πρόγραμμα Σπουδών από προσωπική ανάγκη για την βελτίωση των ακαδημαϊκών μου γνώσεων καθώς το βρήκα πολύ ενδιαφέρον και σκοπός μου είναι η εφαρμογή αυτών στο εργασιακό μου περιβάλλον. Μετά την επιτυχή ολοκλήρωση της ΣΔΥ60 προσδοκώ να έχω βελτιώσει τις γνώσεις μου ως προσ τον προγραμματισμό, την εφαρμογή αυτού στα πεδίο του HCI και λαβώ τις αντίστοιχες γνώσεις για την εφαρμογή του διάχυτου υπολογισμού όστο εν λόγω πεδίο.

**Comment by Katethoma** [comment]
Καλησπέρα. Είμαι απόφοιτος ΤΕΙ Ηλεκτρονικών εδώ και αρκετά χρόνια, εργάζομαι στο Εθνικό Μετσόβιο Πολυτεχνείο σαν ΕΔΙΠ, όπου διεξάγω Εργαστηριακές Ασκήσεις σε φοιτητές σχετικά με το αντικείμενο σπουδών μου, ενώ παράλληλα έχω πολυετή εμπειρία ως ειδικός Ηλεκτρονικής Μικροσκοπίας Σάρωσης σε σχέση με τα υλικά.
Το αντικείμενο του μεταπτυχιακού στην θεματική ενότητα της ΣΔΥ μου φάνηκε πολύ σύγχρονο και ενδιαφέρον. Προσπαθώ να ανταπεξέλθω στις απαιτήσεις του, παρά το γεγονός ότι λόγω επαγγελματικών υποχρεώσεων είχα χάσει την επαφή μου με τον τομέα της Πληροφορικής.

**Comment by NikosAndreadakis** [comment]
Καλησπέρα σας,

Στο [link](https://www.linkedin.com/in/nikolaos-andreadakis-80668513b/) είναι η σελίδα μου στο LinkedIn.

Είμαι πτυχιούχος μηχανολόγος μηχανικός ΤΕΙ. Πάντα είχα πάθος για τις ηλεκτρονικές συσκευές και τον προγραμματισμό μικροελεγκτών. Τελικά, το χόμπι μου κατέληξε επάγγελμα συνδυάζοντας το υπόβαθρό μου σαν μηχανολόγος και τις γνώσεις μου στον προγραμματισμό και πλέον σχεδιάζω ασύρματες συσκευές καταγραφής ενέργειας και διαχείρισης υδάτινων πόρων.

**Comment by gregtsag** [comment]
Καλησπέρα σας,

Στο παρακάτω link η σελίδα μου στο linkedin.

https://www.linkedin.com/in/gregory-tsagris-263b97173

Έχω αποφοιτήσει από το τμήμα Ηλεκτρολόγων μηχανικών και τεχνολογίας υπολογιστών του Πανεπιστημίου Πατρών με εξιδείκευση στα ενεργειακά συστήματα. Εργάζομαι σε βιομηχανία στον τομέα της ενέργειας και στόχος μου από το μεταπτυχιακό αυτό είναι η απόκτηση γνώσεων γύρω από Internet of Things με σκοπό να ασχοληθώ στο μέλλον με το κομμάτι IIoT και OT.

**Comment by Katethoma** [comment]
Καλησπέρα.
Η σελίδα μου στο LinkedIn είναι  
https://www.linkedin.com/in/katerina-thomas-57b49228/

**Comment by atsiano81** [comment]
Καλησπέρα.
Ονομάζομαι Τσιάνος Αντώνιος. Είμαι απόφοιτος του προπτυχιακού τμήματος πληροφορικής του ΕΑΠ. Έχω τελειώσει τις ενότητες 50 και 51 του συγκεκριμένου μεταπτυχιακού. Είμαι αρκετά εξοικιωμένος με την JAVA, τον αντικειμενοστρεφή προγραμματισμό, τα κατανεμημένα  συστήματα και τις τεχνολογίες διαδικτύου. Ομιλώ 3 ξένες γλωσσες Αγγλικά, Ιταλικά και Ισπανικά σε επίπεδα C2, B2 και B2 αντίστοιχα. Εργάζομαι σαν διοικητικός υπάλληλος στο Γ. Ν. Παπαγεωργίου στη Θεσσαλονίκη στο τμήμα νοσηλίων. Είμαι παντρεμένος και πατέρας δύο παιδιών.

**Comment by sotkot** [comment]
Καλησπέρα σε όλους κι από εμένα,

Ονομάζομαι Κοτρώτσιος Σωτήριος.

Περισσότερες πληροφορίες σχετικά με τα επαγγελματικά/ακαδημαϊκά μου ενδιαφέροντα, μπορείτε να βρείτε στο LinkedIn profile μου (είναι up to date μονίμως και αρκετά λεπτομερές):
https://www.linkedin.com/in/sotirios-kotrotsios/

Από την ΣΔΥ60 (όπως κι από όλο το μεταπτυχιακό πρόγραμμα) επιδιώκω να με βοηθήσει να συνεχίσω το ταξίδι μου στα Συστήματα Διάχυτου Υπολογισμού, αποκτώντας επιπρόσθετες γνώσεις (σχεδιαστικές, προγραμματιστικές, αναλυτικές κ.α.) με την ελπίδα αυτές (όπως και η επιστήμη σε γενικότερο πλαίσιο) να εφαρμοστούν με γνώμονα την εξυπηρέτηση των ανθρώπων και την βελτίωση της ποιότητας ζωής τους.

**Comment by vangelidis** [comment]
Καλησπέρα σε όλους,

ονομάζομαι Βασίλης Αγγελίδης και είμαι απόφοιτος του τμήματος πληροφορικής του Τ.Ε.Ι. Θεσσαλονίκης.

Είμαι project manager και υπεύθυνος πληροφορικής του Κέντρου Ελληνικής Νοηματικής Γλώσσας.

Από την ΣΔΥ60 - και γενικότερα από το μεταπτυχιακό πρόγραμμα - προσδοκώ να εξερευνήσω και να αποκτήσω γνώσεις σε τομείς με τους οποίους δεν είχα την ευκαιρία έρθω σε επαφή μέχρι τώρα.

**Comment by vagelisNerantzis** [comment]
Καλησπέρα και από μένα 
Η σελίδα μου στο LinkedIn είναι
https://www.linkedin.com/in/vagelis-nerantzis-b4348b133/

**Comment by gsopy** [comment]
Καλησπέρα,
Είμαι απόφοιτος του τμήματος Ηλεκτρονικών της ΑΣΕΤΕΜ/ΣΕΛΕΤΕ και του τμήματος Διοίκησης Επιχειρήσεων και Οργανισμών του Ε.Α.Π. Ζω και εργάζομαι ως εκπαιδευτικός στη δευτεροβάθμια εκπαίδευση στην Ξάνθη.
